src/pages/NewChatPage.js

```diff
@@ -194,8 +217,14 @@ class NewChatPage extends Component {
         if (userLogins.length < 1) {
             return;
         }
-
-        Report.fetchOrCreateChatReport([this.props.session.email, ...userLogins]);
+        let newChat = {};
+        const chat = this.getChatByParticipants(userLogins);
+        if (!chat) {
+            newChat = ReportUtils.buildOptimisticChatReport(userLogins);
```

Comment:
Why creating a newChat variable when you can just reuse chat?

Reply:
Because the optimistic data is based on whether or not we send an empty `newChat` param https://github.com/Expensify/App/pull/11439/files#diff-8afe5b71ee0436c21364148c86dadd640f2bff3e3d916addbb1f1f6f7e5b6a43R732-R752